### PR TITLE
feature/LWF-2_home-page

### PR DESCRIPTION
**Pull request by JeriRov**

### CODE REVIEW

**Review by borjom1 — APPROVED** [review]
LGTM

File: frontend/src/components/Parallax/Parallax.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        />
+
+        <Image
+          width={500}
+          height={500}
+          ref={link}
+          src="/images/parallax/link.svg"
+          className="link"
+          alt="link"
+        />
```
Comment (milkanuch) [inline frontend/src/components/Parallax/Parallax.tsx:140]: use list and map, or move into another component

File: frontend/src/components/Parallax/Parallax.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      tl.to(
+        btn.current,
+        {
+          opacity: 1,
+        },
+        1
+      );
+    });
+    return () => ctx.revert();
+  }, []);
```
Comment (milkanuch) [inline frontend/src/components/Parallax/Parallax.tsx:95]: create a hook that implements this logic, or move it into functions, please.

File: frontend/src/components/CustomButton/CustomButton.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        type="button"
+        className={`bg-black text-white px-8 py-4 mt-3 font-bold rounded-lg ${className}`}
+        ref={ref}
+        {...props}
+      >
+        {children}
+      </button>
+    );
+  }
+);
```
Comment (milkanuch) [inline frontend/src/components/CustomButton/CustomButton.tsx:21]: The best way to handle a custom button is to add a small debounce on click.

[More Information](https://sandny.com/2021/02/16/usidebounce-with-react-hooks/)

**Pull request merged**